dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/bannersandmessages/banners/BannerLimits.java

```diff
@@ -1,9 +1,9 @@
 package edu.harvard.iq.dataverse.bannersandmessages.banners;
 
 import javax.ejb.Local;
-import javax.ejb.Stateful;
+import javax.ejb.Stateless;
 
-@Stateful
+@Stateless
 @Local
```

Comment:
I don't think that `@Local` annotation is needed

Reply:
Done